src/tests/ftest/util/agent_utils_params.py

```diff
@@ -33,14 +33,12 @@ class DaosAgentTransportCredentials(TransportCredentials):
     def __init__(self):
         """Initialize a TransportConfig object."""
         super(DaosAgentTransportCredentials, self).__init__(
-            "/run/agent_config/transport_config/*", "transport_config")
-
-        # Additional daos_agent transport credential parameters:
-        #   - server_name: <str>, e.g. "daos_server"
-        #       Name of server accodring to its certificate [daos_agent only]
-        #
-        self.server_name = BasicParameter(None, "daos_server")
+            "/run/transport_config/*", "transport_config")
 
+        self.allow_insecure = BasicParameter(False, False)
+        self.ca_cert = BasicParameter(None, "./daosCA/certs/daosCA.crt")
+        self.cert = BasicParameter(None, "./daosCA/certs/agent.crt")
+        self.key = BasicParameter(None, "./daosCA/certs/agent.key")
```

Comment:
These paths likely will not work with RPM testing.  As mentioned before you should test this PR with "RPM-test: true" in your final commit also to ensure you are not breaking testing-with-RPMs.

Reply:
I don't understand why you resolved this conversation without doing anything to address it.  This is why it's the responsibility of the person who started the conversation to resolve them.